data/data_parser.py

```diff
@@ -57,8 +57,10 @@
                     help='The path to the dat file from uniprotKB TrEMBL.')
 parser.add_argument('--filepath', type=str,
                     help='The path to the input file.')
-parser.add_argument('--accession', type=str,
-                    help='ENCODE or IGVF file accession to fetch and parse data from.')
+parser.add_argument('--accessions', nargs='+', type=str,
+                    help='One or more ENCODE or IGVF file accessions to fetch and parse data from.')
+parser.add_argument('--replace', action='store_true',
```

Comment:
I assume we should use replace = false in most cases (maybe add a default = false?), and will need to remember to fix conflicting jsonl files on s3 if replace = true.